### DISCUSSION TITLE
Can this be used in a purely offline (no internet) setting?

### DISCUSSION THREAD

**Discussion by Jibril2** [flashlight/flashlight#659] [Q&A]
Can this be used in a purely offline (no internet) setting? I want to embed ASR into the pipeline of a project I am working on, but it needs to be able to work on a device with no access to any internet after installation.

**Comment by Grottoh** [comment]
Yeah, it can be used offline as well (as far as I'm aware without any sacrifices).

**Reply by Jibril2** [reply]
Amazing! Now if I can only get it to install. It worked fantastic in the collab notebook but running into tons of issues on my local install. Once I get that up though, this is incredibly exciting.
